v4/source/introduction.yaml

```diff
@@ -642,6 +642,6 @@ x-tagGroups:
       - exports
       - usage
 servers:
-  - url: http://your-mattermost-url.com/api/v4
-  - url: https://your-mattermost-url.com/api/v4
+  - url: http://your-mattermost-url.com
+  - url: https://your-mattermost-url.com
```

Comment:
This is the "bit that counts" relative to what we wanted to fix. Note that Playbooks had its own servers, but they got clobbered with the merge.